paymentapi/src/test/java/fi/hel/verkkokauppa/payment/api/PaymentAdminControllerTest.java

```diff
@@ -147,14 +171,230 @@ public void savePaymentFilter() throws JsonProcessingException {
         Assertions.assertNotNull(fourthExpectedFilterDto.getCreatedAt());
 
         // remove test filters from database
-        idsToDelete.add(getActualFilterId(orderPaymentFilterForNordea));
-        idsToDelete.add(getActualFilterId(orderPaymentFilterForOp));
-        idsToDelete.add(getActualFilterId(merchantPaymentFilterForNordea));
-        idsToDelete.add(getActualFilterId(merchantPaymentFilterForOp));
+        paymentFiltersToBeDeleted.add(getActualFilterId(orderPaymentFilterForNordea));
+        paymentFiltersToBeDeleted.add(getActualFilterId(orderPaymentFilterForOp));
+        paymentFiltersToBeDeleted.add(getActualFilterId(merchantPaymentFilterForNordea));
+        paymentFiltersToBeDeleted.add(getActualFilterId(merchantPaymentFilterForOp));
+    }
+
+    /*
+     * It tests the create payment method endpoint.
+     */
+    @Test
+    @RunIfProfile(profile = "local")
+    public void whenCreatePaymentMethodWithValidData_thenReturnStatus201() {
```

Comment:
No _ underlines in methods.

Reply:
I'll make the change across the tests that I have added.